File: ApiSuvesaPos/Negocio/Modelo/cxc/devoluciones_ventas/devoluciones_ventas.vb
Namespace Modelo
    Public Class devoluciones_ventas
        Public Property Devolucion As Long
        Public Property Id_Factura As Long
        Public Property SaldoAnt_Fact As Decimal
        Public Property SubTotalGravado As Decimal
        Public Property SubTotalExcento As Decimal
        Public Property Descuento As Decimal
        Public Property Impuesto As Decimal
        Public Property Monto As Decimal
        Public Property Fecha As DateTime
        Public Property Contabilizado As Boolean
        Public Property NumeroAsiento As Long
        Public Property Anulado As Boolean
        Public Property Cedula_Usuario As String
        Public Property Cod_Moneda As Integer
        Public Property Transporte As Decimal
        Public Property ContabilizadoCosto As Boolean
        Public Property NumeroAsientoCosto As Long
        Public Property caja As Integer
        Public Property CONSECUTIVO As Long
        Public Property ENVIADODGT As Boolean
        Public Property ESTADODGT As String
        Public Property CONSECUTIVODGT As String
        Public Property CLAVEDGT As String
        Public Property MontoDevolucion As Decimal
        Public Property Num_Apertura As Integer
        Public Property UsuarioRecibio As String
        Public Property NotasDevolucion As String
        Public Property Articulos_Ventas_Devueltos As New Modelo.articulos_ventas_devueltos

    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxc/devoluciones_ventas/devoluciones_ventas.cs
using System;

namespace Modelo
{
    public class devoluciones_ventas
    {
        public long Devolucion { get; set; }
        public long Id_Factura { get; set; }
        public decimal SaldoAnt_Fact { get; set; }
        public decimal SubTotalGravado { get; set; }
        public decimal SubTotalExcento { get; set; }
        public decimal Descuento { get; set; }
        public decimal Impuesto { get; set; }
        public decimal Monto { get; set; }
        public DateTime Fecha { get; set; }
        public bool Contabilizado { get; set; }
        public long NumeroAsiento { get; set; }
        public bool Anulado { get; set; }
        public string Cedula_Usuario { get; set; }
        public int Cod_Moneda { get; set; }
        public decimal Transporte { get; set; }
        public bool ContabilizadoCosto { get; set; }
        public long NumeroAsientoCosto { get; set; }
        public int caja { get; set; }
        public long CONSECUTIVO { get; set; }
        public bool ENVIADODGT { get; set; }
        public string ESTADODGT { get; set; }
        public string CONSECUTIVODGT { get; set; }
        public string CLAVEDGT { get; set; }
        public decimal MontoDevolucion { get; set; }
        public int Num_Apertura { get; set; }
        public string UsuarioRecibio { get; set; }
        public string NotasDevolucion { get; set; }
        public Modelo.articulos_ventas_devueltos Articulos_Ventas_Devueltos { get; set; } = new Modelo.articulos_ventas_devueltos();
    }
}
